MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/snapshotter/MapSnapshotter.java

```diff
@@ -57,7 +79,14 @@
      * @param snapshot the snapshot
      */
     void onSnapshotReady(MapSnapshot snapshot);
+  }
+
+  public interface MapSnapshotterObserver {
+    void onDidFailLoadingStyle(MapSnapshotter snapshotter, String reason);
```

Comment:
I didn't like that we need to pass snapshotter in every method, maybe https://github.com/mapbox/mapbox-gl-native-android/commit/ab2d2b67848cabfe0fd0d18a9c003b86974342fc would be better? @Chaoba @tobrun?

Reply:
I am think about not adding this new interface as developers don't need to be notified whether style is loaded or not. What they need to do is set up the style and start snapshotter then waiting for the result. The previous interface  `SnapshotReadyCallback` and `ErrorHandler` could meet the requirements.